parsers/s01-parse/crowdsecurity/postfix-logs.yaml

```diff
@@ -58,4 +58,38 @@ statics:
       expression: "evt.Parsed.remote_host"
     - meta: log_type
       value: postfix
-
+---
+onsuccess: next_stage
+filter: "evt.Parsed.program == 'postfix/cleanup'"
+name: krvtz/postfix-spam
+description: "Parse postfix logs"
+nodes:
+  - grok:
+      apply_on: message
+      pattern: 'milter-reject: END-OF-MESSAGE from %{POSTFIX_HOSTNAME:remote_host}\[%{IP:remote_addr}\]: 4.7.1 Spam message rejected'
+      statics:
+        - meta: log_type_enh
+          value: spam-attempt
+  - grok:
+      apply_on: message
+      pattern: ' %{POSTFIX_HOSTNAME:remote_host}\[%{IP:remote_addr}\]: 450 4.1.1 <[a-zA-Z0-9@-]+>: Recipient address rejected'
+      statics:
+        - meta: log_type_enh
+          value: spam-attempt
+  - grok:
+      apply_on: message
+      pattern: ' %{POSTFIX_HOSTNAME:remote_host}\[%{IP:remote_addr}\]: SASL PLAIN authentication failed'
+      statics:
+        - meta: log_type_enh
+          value: posfix_failed_auth
```

Comment:
Can this value be `spam-attempt` to match original parser so we dont need to make an additional scenario. However, does a single login failure cause 2 of these logs?

Reply:
Resolved by removal of this rule